workbench_access.permissions.yml

```diff
@@ -32,3 +32,8 @@ view workbench access information:
   description: 'View the editorial access status of each content page.'
   title: 'View Workbench Access information'
   restrict access: FALSE
+
+'allow taxonomy term delete':
```

Comment:
My temptation here is to move all of this into a submodule that simply disables deletes entirely if they are used for access control. We should be able to apply that to all entity types used for access control without having entity-specific handling in our code.